source/assets/public/index.css

```diff
@@ -274,26 +293,10 @@ body {
   margin-top: -1rem;
 }
 
-
-
-
 .footer {
   margin-bottom: 6rem;
-  width: 90%;
```

Comment:
You can see what I mean when you compare the centring of the footer to the centring of the sponsored elements.